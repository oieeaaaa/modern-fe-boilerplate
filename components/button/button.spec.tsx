import render, { screen } from 'test-utils'
import Button from '.'

describe('Button', () => {
  it('should pass', () => {
    render(<Button>Yo</Button>)

    expect(screen.getByText('Yo')).toBeInTheDocument()
    expect(screen.getByRole('button')).toMatchSnapshot()
  })
})
