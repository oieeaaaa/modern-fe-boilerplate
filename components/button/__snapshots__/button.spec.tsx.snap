// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should pass 1`] = `
<button
  class="button"
>
  Yo
</button>
`;
